go/pkg/cas/upload.go

```diff
@@ -651,6 +651,16 @@ func (u *uploader) visitRegularFile(ctx context.Context, absPath string, info os
 		// advantage of ByteStream's built-in presence check.
 		// https://github.com/bazelbuild/remote-apis/blob/0cd22f7b466ced15d7803e8845d08d3e8d2c51bc/build/bazel/remote/execution/v2/remote_execution.proto#L250-L254
 
+		if res, err := u.findMissingBlobs(ctx, []*uploadItem{item}); err != nil {
+			return nil, errors.Wrapf(err, "failed to check existence")
+		} else if len(res.MissingBlobDigests) == 0 {
```

Comment:
nit: I wonder "if" is better than "else if" since it's not an error case.

```
res, err := u.findMissinBlobs(...)
if err != nil {
  ...
}
len(res.MissingBlobDigests) == 0 {
  ...
}
```

Reply:
I prefer to have narrower scope for variables here.

Reply:
I see, ok.